profile/models/c5g7/c5g7-cmfd.cpp

```diff
@@ -465,8 +465,8 @@ int main() {
   Cmfd cmfd;
   cmfd.setSORRelaxationFactor(1.5);
   cmfd.setLatticeStructure(51, 51);
-  int cmfd_group_structure[3] = {1,4,8};
-  cmfd.setGroupStructure(cmfd_group_structure, 3);
+  std::vector< std::vector<int> > cmfd_group_structure = {{1,4,7}};
```

Comment:
@zechzhao if you revise to the recommendation by @geogunow above we will merge in your PR right away.